modules/manage/pages/kubernetes/troubleshooting/k-troubleshoot.adoc

```diff
@@ -456,6 +456,44 @@ Error: UPGRADE FAILED: cannot patch "redpanda-console" with kind Deployment: Dep
 
 To fix this error, set `console.enabled` to `false` so that Helm doesn't try to deploy Redpanda Console again.
 //end::deployment-cannot-patch[]
+
+//tag::pending-rollback[]
+=== Helm is in a pending-rollback state
+
+An interrupted Helm upgrade process can leave your Helm release in a `pending-rollback` state. This state prevents further actions like upgrades, rollbacks, or deletions through standard Helm commands. To fix this issue, do the following:
+
+. Identify the Helm release that's in a `pending-rollback` state:
++
+[source,bash]
+----
+helm list --namespace <namespace> --all
+----
++
+Look for releases with a status of `pending-rollback`. These are the ones that need intervention.
+
+. Verify the Secret's status to avoid affecting the wrong resource:
++
+[source,bash]
+----
+kubectl --namespace <namespace> get secret --show-labels
+----
++
+Identify the Secret associated with your Helm release by its `pending-rollback` status in the labels.
++
+WARNING: Ensure you have correctly identified the Secret to avoid unintended consequences. Deleting the wrong Secret could impact other deployments or services.
+
+. Delete the Secret to clear the `pending-rollback` state:
++
+[source,bash]
+----
+kubectl --namespace <namespace> delete secret -l status=pending-rollback
+----
+
+After clearing the `pending-rollback` state:
+
+* *Retry the upgrade*: Start the upgrade process again. Investigate the initial failure to avoid repeating the same mistake.
```

Comment:
```suggestion
* *Retry the upgrade*: Restart the upgrade process. You should investigate the initial failure to avoid getting into the `pending-rollback` state again.
```